test/executor.test.js

```diff
@@ -61,13 +62,18 @@ function testRightOrder (d, done) {
             d.update({ a: 2 }, { a: 3 }, {}, function () {
               d.find({}, function (err, docs) {
                 docs[0].a.should.equal(3);
-                done();
 
+                process.removeAllListeners('uncaughtException');
+                for (var i = 0; i < currentUncaughtExceptionHandlers.length; i += 1) {
+                  process.on('uncaughtException', currentUncaughtExceptionHandlers[i]);
+                }
```

Comment:
Cleaning up the listener added during the test and adding back any of the previously attached listeners.